setup.py

```diff
@@ -13,17 +13,23 @@
     url='https://github.com/edx/edx-organizations',
     license='AGPL',
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 5 - Production/Stable',
         'Environment :: Web Environment',
+        'Framework :: Django',
+        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0',
+        'Framework :: Django :: 2.1',
```

Comment:
can we truly say this here, seeing that we "pin" django to <2.0?

Reply:
In this repo, tox testing is performed on Django 2.0/2.1 - so seems fine:
https://github.com/edx/edx-organizations/blob/master/tox.ini